conn.go

```diff
@@ -25,6 +25,12 @@ type message struct {
 	To []string
 }
 
+type ConnInfo struct {
```

Comment:
Last one and I stop bothering you: possible to rename to `ConnectionState` for consistency with the `tls` package? :P

Reply:
Ok.